src/components/emoji/nimble-emoji.js

```diff
@@ -93,7 +93,11 @@ class NimbleEmoji extends React.PureComponent {
 
     let data = this._getData(this.props)
     if (!data) {
-      return null
+      if (this.props.fallback) {
+        return this.props.fallback(null)
```

Comment:
@jgfidelis do you think we should replicate current master of `emoji-mart` and pass back the props too in both these fallback calls?

Reply:
@pederjohnsen yes, I think it would be nice, I was thinking about doing it on this PR but I did not know if you would like hehe. If you are open to the idea, it could be `this.props.fallback(data, this.props)`

Reply:
@jgfidelis yeah sounds good :+1: